src/man/sssd-ad.5.xml

```diff
@@ -75,6 +75,12 @@
             sudo and autofs provider. No configuration of the access provider
             is required on the client side.
         </para>
+        <para>
+            Note that both auth and access AD providers rely on the
+            id_provider being set to "ad" as well. If you wish to use a
```

Comment:
The wording here makes sense to me but I'm afraid it might be a bit confusing to some admins, I would propose something like:

If `auth_provider = ad` or `access_provider = ad` is configured in sssd.conf then the `id_provider` must also be set to "ad"